cfme/tests/services/test_service_customer_bz.py

```diff
@@ -163,3 +165,29 @@ def test_update_dynamic_checkbox_field(appliance, import_datastore, import_data,
             "checkbox").checkbox.read() is True and not view.submit_button.disabled, timeout=60)
     except TimedOutError:
         pytest.fail("Checkbox did not checked and Submit button did not enable in time")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1720617)])
```

Comment:
**Suggestion:** 
@niyazRedhat you can use instead: `@pytest.mark.meta(automates=[1720617])`

Reply:
Ignore this as @ganeshhubale Already reviewed.